### PR TITLE
Integrate common utils

### PR DESCRIPTION
**Pull request by calina-c**
Towards #226.

### CODE REVIEW

**Review by trentmc — DISMISSED** [review]
I did a lightweight review. Looks sane to me. 

I did not run the tests, for this round I'm assuming that you've got them passing. 

This PR pulls test coverage down. As discussed in slackm I'm assuming that a new issue will be created that adds more tests in order to get test coverage back up.

File: ocean_lib/common/__init__.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#
+# Copyright 2021 Ocean Protocol Foundation
+# SPDX-License-Identifier: Apache-2.0
+#
+__version__ = "0.4.2"
```
Comment (ssallam) [inline ocean_lib/common/__init__.py:5]: whats the use of version here?

Comment (calina-c) [reply ocean_lib/common/__init__.py]: I forgot to remove this, fixed now :)

Comment (calina-c) [reply ocean_lib/common/__init__.py]: > I did a lightweight review. Looks sane to me.
> 
> I did not run the tests, for this round I'm assuming that you've got them passing.
> 
> This PR pulls test coverage down. As discussed in slackm I'm assuming that a new issue will be created that adds more tests in order to get test coverage back up.

@trentmc added #292.

**Pull request merged**